### PR TITLE
Improve output of init command

### PR DESCRIPTION
**Pull request by rosen-vladimirov**
Add message that project has been successfully initialize after calling '$ appbuilder init' or '$ appbuilder init hybrid'. Successfully initialized %s project. (%s being Cordova, NativeScript or MobileWebsite).

http://teampulse.telerik.com/view#item/285141

### CODE REVIEW

File: lib/commands/project.ts [hunk 1]
```diff
@@ -196,13 +203,13 @@ export class InitCommand extends InitProjectCommandBase {
...
-				this.$logger.info("Attempting to initialize Apache Cordova project.");
+				this.$logger.info("Attempting to initialize Cordova project.");
 				this.initializeProjectFromExistingFiles(this.$projectConstants.TARGET_FRAMEWORK_IDENTIFIERS.Cordova).wait();
 			} else if(this.isProjectType("NativeScript").wait()) {
-				this.$logger.info("Attempting to initialize  NativeScript project.");
+				this.$logger.info("Attempting to initialize NativeScript project.");
 				this.initializeProjectFromExistingFiles(this.$projectConstants.TARGET_FRAMEWORK_IDENTIFIERS.NativeScript).wait();
 			} else if(this.isProjectType("Mobile Website").wait()) {
-				this.$logger.info("Attempting to initialize Mobile Website project.");
+				this.$logger.info("Attempting to initialize MobileWebsite project.");
```
Comment (teobugslayer) [inline lib/commands/project.ts]: Why this change?

Comment (rosen-vladimirov) [inline lib/commands/project.ts]: Because new output is "Successfully initialized MobileWebsite project.". It looks strange in the following way:
Attempting to initialize Mobile Website project.
Successfully initialized MobileWebsite project.

As the code for successfull initialization is using TARGET_FRAMEWORK_IDENTIFIERS, which is string, I decided to change it here to make the output consistent

**Comment by teobugslayer** [comment]
:+1:
